tests/regression/test_mean_error.py

```diff
@@ -92,6 +93,20 @@ def test_mean_error_functional(self, preds, target, sk_metric, metric_class, met
             sk_metric=partial(sk_metric, sk_fn=sk_fn),
         )
 
+    @pytest.mark.skipif(
+        not _TORCH_GREATER_EQUAL_1_6,
```

Comment:
Not sure if we really should skip here or show a more meaningful error message.

Reply:
@SkafteNicki ^^

Reply:
@justusschock you got any proposal?

Reply:
@justusschock more meaningful message in test or the class itself? wouldn't be easier to test for half precision tensors in the base metric class and check if torch version >= 1.6